EdgeMultiplay/Scripts/DataContracts.cs

```diff
@@ -490,6 +501,28 @@ public class Player
         /// </summary>
         [DataMember]
         public int playerAvatar = 0;
+
+        /// <summary>
+        /// helper instance variable for serializing and deserializing playerTags
+        /// </summary>
+   		[DataMember (EmitDefaultValue = false)]
```

Comment:
nit: extra tab

Reply:
This tab is only on here in the PR, that is weird